### PR TITLE
Add normalizer-functions docs and format navigation at all data components

### PR DESCRIPTION
**Pull request by pablocasares**

### CODE REVIEW

**Review by andresgomezfrr — COMMENTED** [review]
Change `Normalizer Functions` to `Normalizer Functions Repository`

Change the link inside normalizer to new docs. Currently, the link goes to `https://wizzie-io.github.io/normalizer-functions/`

**Comment by pablocasares** [comment]
:+1 done!

**Comment by Chorro** [comment]
In all normalizer versions, the links to **FileBootstrapper** and **KafkaBootstrapper** don't work

**Comment by pablocasares** [comment]
@Chorro done! thanks!

**Comment by Chorro** [comment]
For all normalizer versions. In `Custom Functions` section there are broken links:
- `SimpleMapper`
- `MaxValueMapper`
- `ClassificationMapper`

**UPDATE**:
The link to `MapperFunction class` also is broken

**UPDATE2**:
`Functions` link also is broken

**Comment by Chorro** [comment]
There is a broken link in enricher docs. In `Base Configuration` section there is a table with enricher's properties, the row with `multi.id` has a broken link to `Multi Tenant` section. There is a missing point between `tenant` and `html`:

image

**Comment by Chorro** [comment]
There are some wrong image's references in `Enrichment Query Language` section. Could you please review it?:

image

**Comment by Chorro** [comment]
In ZZ-CEP docs, in `Docker Configuration` section the `base tutorial` link does reference to normalizer base tutorial:

image

**Comment by pablocasares** [comment]
@Chorro done all

**Comment by Chorro** [comment]
Ok @pablocasares 👍 . Thanks 😄

**Comment by Chorro** [comment]
The images of `mobility` section (`GeoTaggingMapper` function) aren't available in `Normalizer Function Repository` docs. Could you fix it?

image

**UPDATE**:

I get the next errors:

```
[2019-01-11 08:13:26] ERROR `/assets/images/geotagging/radius.svg' not found.
[2019-01-11 08:13:26] ERROR `/assets/images/geotagging/rectangle.svg' not found.
[2019-01-11 08:13:26] ERROR `/assets/images/geotagging/polygon.svg' not found.
[2019-01-11 08:13:26] ERROR `/assets/images/geotagging/superposition.svg' not found.
```

**Comment by pablocasares** [comment]
@Chorro done, thanks again! 😄

**Comment by pikislabis** [comment]
When rendering a link as done in the attached screenshot, the link is usually open in a new tab. Those links in the screenshot are open in the same browser tab.

screenshot-127 0 0 1-4000-2019 01 11-10-45-25

File: _data/sidebars/normalizer_0_7_2_sidebar.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ entries:
       url: /normalizer_0.7.2_debug_functions.html
       output: web
     - title: Functions Repository
-      url: https://wizzie-io.github.io/normalizer-functions/
+      url: https://wizzie-io.github.io/normalizer-functions_latest_index.html
```
Comment (andresgomezfrr) [inline _data/sidebars/normalizer_0_7_2_sidebar.yml]:
```suggestion
      url: /normalizer-functions_latest_index.html
```

I think that this link can be relative

Comment (pablocasares) [reply _data/sidebars/normalizer_0_7_2_sidebar.yml]: done!, thanks! 😄

**Comment by pablocasares** [comment]
@pikislabis done, thanks! 😄

**Review by andresgomezfrr — APPROVED** [review]

**Review by pikislabis — APPROVED** [review]

**Pull request merged**